src/hooks/useLoginUrl/__tests__/useLoginUrl.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { renderHook } from '@testing-library/react-hooks';
+import useLoginUrl from '..';
+
+describe('Login', () => {
+  it('Should return a string containing oauth2', () => {
+    const { result } = renderHook(() => useLoginUrl('en'));
+    expect(result.current.getUrl()).toContain('/oauth2/');
+  });
+  it('Should return a string containing language "en"', () => {
+    const { result } = renderHook(() => useLoginUrl('en'));
+    expect(result.current.getUrl()).toContain('/oauth2/');
+  });
```

Comment:
please add test cases to check the `app_id`, `login_url` as well.

Reply:
Done